x-pack/metricbeat/module/mssql/log/data.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package db
+
+import (
+	"time"
+
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
```

Comment:
File included for clarity but it's not being used.

Reply:
If for any reason we want to know to aspect of the data that is being sent to ES, we tend to go to look the `data.go` file. Of course, we can also look at the `data.json` generated file but there are not guarantees that it's updated (neither this file).

In short, it's to help any maintainer that it's not me :)

Reply:
`data.go` uses to contain the code for mapping the data to events, if there is no clear separation for that in this metricset I'd prefer to remove this file than to keep it with unused code.

If it is to know more about the generated data, as you mention `data.json` can be used.

Reply:
File has been removed :+1: